.github/workflows/ci.yml

```
@@ -32,6 +32,34 @@ jobs:
               with:
                   useConfigFile: true
 
+            - name: Set up Python
+              uses: actions/setup-python@v5
+              with:
+                python-version: "3.11"
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 18 but found 16

(indentation)